share/src/main/resources/assets/wdl/lang/zh_CN.lang

```diff
@@ -0,0 +1,523 @@
+# World Downloader translation file
+# All of the text on the left side of the = is the key, while the right side is the value.
+# '%s' and similar are formatting tokens, as described on the formatter
+# docs: https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html
+# Only '%s' and '%%' are allowed.  '%s' is a string, and '%%' is an escaped percent.
+# You can reference an argument at a specific index with %i$s, where 'i'
+# is the number.
+# Everything after a '#' is a comment.
+#
+# For description messages in GUIs, you are also allowed to use \n to indicate a new line.
+# This cannot be escaped.
+#
+# Additionally, Minecraft's §-based color codes may be used.  See
+# https://minecraft.gamepedia.com/Formatting_codes for more info.
+
+# Translator name, preferably in the form of 'Translated to %s by <your name>'.
+# '%s' will be replaced with the language name automatically.
+# If left blank, it will not be displayed.
+wdl.translatorCredit=
```

Comment:
Do you want to fill this one in?  It's displayed in the about screen.

Reply:
alright